core-build/gulp-core-build-sass/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@microsoft/gulp-core-build-sass",
-  "version": "4.6.34",
+  "version": "4.6.35",
```

Comment:
Don't manually bump the version. When Rush publishes this package, it will automatically bump the version based on the changefile.

Reply:
Removed manual version bump.